### PR TITLE
Update latest-dev Management Center version to 4.2021.04-SNAPSHOT

### PR DESCRIPTION
**Pull request by emre-aydin**

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # Redirect latest-dev imdg alias to the latest-dev version
 /imdg/latest-dev/*  /imdg/4.2-snapshot/:splat 200!
 # Redirect latest-dev management-center alias to the latest version
-/management-center/latest-dev/*  /management-center/4.2021.02-snapshot/:splat 200!
+/management-center/latest-dev/*  /management-center/4.2021.04-SNAPSHOT/:splat 200!
```
Comment (JakeSCahill) [inline _redirects:8]:
```suggestion
/management-center/latest-dev/*  /management-center/4.2021.04-snapshot/:splat 200!
```

**Review by JakeSCahill — CHANGES_REQUESTED** [review]
`snapshot` needs to be lowercase like the original file

**Review by JakeSCahill — APPROVED** [review]
👍

**Comment by JakeSCahill** [comment]
The build is failing because the `antora.yml` in the tagged `v4.2021.02` has not been updated with the new `verion`

**Comment by emre-aydin** [comment]
@JakeSCahill I have updated it now.

**Comment by emre-aydin** [comment]
https://github.com/hazelcast/management-center-docs/blob/v4.2021.02/docs/antora.yml#L3

**Pull request merged**